basic_pipeline/4_Caps.md

```diff
@@ -0,0 +1,116 @@
+# Introduction
+We owe you something...and we would like to pay it back as soon as possible!
+As promised in the [3rd chapter](3_Source.md), we will talk more about the concept of caps - which in fact we have used in the previous chapter, but which weren't described in the sufficient way there.
+# What are caps?
+Caps (an abbreviation of the *capabilities*) is a concept allowing us to define what kind of data is flowing through the pad. 
+In the Membrane Framework's nomenclature, we say, that we define a caps specification for a given element.
+
+I believe that an example might speak here louder than a plain definition, so I will try to describe the caps with the real-life scenario example.
```

Comment:
I would like us to stick to plural narrator, replacing "I will try to ..." with "we will try to ..."

Reply:
done